### ISSUE TITLE
idleinhibit visible

### ISSUE THREAD

**Issue by Kommynct** [hyprwm/Hyprland#5010]
### Description

Sway allows you to inhibit idle when a window is visible, is there any equivalent? the documentation doesn't seem to have one.

**Comment by tharrmeehan** [comment]
Are you talking about this?

![grafik](https://github.com/hyprwm/Hyprland/assets/58373200/9ae8a88a-ea0e-402a-87b6-cfb1c4a96652)

https://wiki.hyprland.org/Configuring/Window-Rules/

**Comment by Kommynct** [comment]
Yes, there is no "visible" option there, so I assume that has not been added but I'm not very familiar with this

**Comment by tharrmeehan** [comment]
Wouldn't the 'focus' mode suffice for that? E.g. if a window is focused, the idleinhibit would be active and otherwise not. Correct me if I'm wrong though.

**Comment by Kommynct** [comment]
No, it would not, if I have the window open on the second screen, and i'm doing things on the first screen, I still want the inhibit idle to be active.

**Comment by tharrmeehan** [comment]
This is what I use for the use-case you described (for a specific application) and it works for me, if zoom is open/active then idleinhibit is active. No matter the monitor / workspace.

![grafik](https://github.com/hyprwm/Hyprland/assets/58373200/2945447b-ecc9-418a-9869-10c83d067221)

**Comment by Kommynct** [comment]
The problem with that is that if I forget about the window, because it's open in the background my computer doesn't suspend

**Comment by vaxerski** [comment]
Hello there!

This issue has been closed, as we are moving from issues to discussions. More about this can be found [here](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) and in #9854.

Firstly, please make sure this issue is still relevant in latest Hyprland. If so, we ask you to open a [discussion](https://github.com/hyprwm/Hyprland/discussions) (please read the discussion guidelines first, in the pinned post)

Thank you for your understanding!
